docs/getting-started/net-map-req.asciidoc

```diff
@@ -11,6 +11,11 @@ Depending on your {kib} setup, to display and interact with data on the
 NOTE: To see source and destination connections lines on the map, you must
 configure `source.geo` and `destination.geo` ECS fields for your indices.
 
+[float]
+[[prereq-perms]]
+=== Permissions required
+In order to view the map, you need at least "Read" privileges for `Maps`. To configure it, you need "All" privileges. Maps privilege settings are under *Kibana privileges* -> *Analytics* -> *Maps*.
```

Comment:
Per other pages concerning role privs (like [this](https://www.elastic.co/guide/en/security/8.0/detections-permissions-section.html)), backticks for the privilege options:
```suggestion
In order to view the map, you need at least `Read` privileges for `Maps`. To configure it, you need `All` privileges. Maps privilege settings are under *Kibana privileges* -> *Analytics* -> *Maps*.
```